cumulus/parachain-template/node/src/service.rs

```diff
@@ -202,6 +202,13 @@ where
 		return Err("Light client not supported!".into())
 	}
 
+	let code_executor = sc_executor::NativeElseWasmExecutor::<Executor>::new(
```

Comment:
Isn't it already instantiated somewhere automatically though?

Reply:
Yes, it was instantiated above https://github.com/subspace/subspace/blob/7a8f22df558f775e18c2dc619f9c71455717cec4/cumulus/parachain-template/node/src/service.rs#L101-L114, but it's not exposed later once consumed, hence I simply reconstruct it.

Reply:
You can clone it though, would be cleaner than creating new instance from scratch, WDYT?

Reply:
Not cloneable :D https://github.com/subspace/substrate/blob/8954904b4a8483f70de0afdf7783cd89861c58ff/client/executor/src/native_executor.rs#L336

Reply:
Actually it is: https://github.com/subspace/substrate/blob/8954904b4a8483f70de0afdf7783cd89861c58ff/client/executor/src/native_executor.rs#L609

At least it is not in a different file :wink:

Reply:
Nice catch! Totally missed that.